src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +57,46 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            String date = dateParams.length != 0
+                    ? LocalDate.of(dateParams[0], dateParams[1], dateParams[2]).toString()
+                    : "";
+            return parseDateByFormatter(date, FORMATTER);
```

Comment:
```suggestion
            return LocalDate.of(dateParams[0], dateParams[1], dateParams[2]).toString();
```
і ще дещо треба буде змінити в catch

Reply:
Done